kodilla-spring/src/main/java/com/kodilla/spring/calculator/Calculator.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.spring.calculator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Calculator {
+
+    @Autowired
+    private Display display;
+
+    public double add(double a, double b) {
+        System.out.print(a + " + " + b);
```

Comment:
a po kiego wuja to? Masz przeciez obiekt display?

Reply:
Display wyświetla wartość końcową, już po po obliczeniu. Tutaj dodałem sout, żeby było widać działanie